src/pages/send-bulk-message.js

```diff
@@ -0,0 +1,216 @@
+import React from 'react';
+import Layout from '../components/layout';
+import Link from 'next/link';
+import { useState, useEffect } from 'react';
+import { Button, Checkbox, Dropdown } from '../components/Form';
+import { CallbackGateway } from '../gateways/callback';
+import { useRouter } from 'next/router';
+import {PRE_CALL_MESSAGE_TEMPLATE} from '../helpers/constants';
+import { GovNotifyGateway  } from "../gateways/gov-notify";
+import { unsafeExtractUser } from "../helpers/auth";
+import { callTypes } from "../helpers/constants";
+
+export default function AssignCallsPage() {
+    const router = useRouter();
+		const user = unsafeExtractUser()
+    const [assignedCallbacks ,setAssignedCallbacks]  = useState(null)
+    const [unAssignedCallbacks ,setUnAssignedCallbacks]  = useState(null)
+    const [previewTemplate, setTemplatePreview] = useState([])
+    const [unassigned, setUnassigned] = useState({
+      value:false,
+      sendSms:false
+    })
+    const [assigned, setAssigned] = useState({
+      value:false,
+      sendSms:false
+    })
```

Comment:
I'd probably have this as one object, something like this:

```const [selectedCases, setSelectedCases] = useState({
        assigned:false,
        unassigned:false
      })
 ```
 
 And update their values if they get selected.
 This also works so you don't need to change it tho

Reply:
the reason I have it like this is so thats its easier to add emails in the future